client/transaction-pool/src/lib.rs

```diff
@@ -49,6 +49,23 @@ pub struct BasicPool<PoolApi, Block>
 		PoolApi: sc_transaction_graph::ChainApi<Block=Block, Hash=Block::Hash>,
 {
 	pool: Arc<sc_transaction_graph::Pool<PoolApi>>,
+	api: Arc<PoolApi>,
+	revalidation_strategy: Arc<Mutex<RevalidationStrategy<NumberFor<Block>>>>,
+}
+
+/// Type of revalidation.
+pub enum RevalidationType {
+	/// Light revalidation type.
+	///
+	/// During maintaince, transaction pool makes periodic revalidation
```

Comment:
I guess you mean that the entire pool is being revalidated, right?
Cause currently the docs for Light and Full are not super clear about differences, If I didn't know exactly how it behaved before the changes I would have hard time deciding which one to choose.

Maybe add some more details on this logic?

Reply:
Added :)